scripts/run-e2e-job.sh

```diff
@@ -5,7 +5,7 @@ set -o nounset
 set -o pipefail
 
 ROOT_DIR=$(dirname "${BASH_SOURCE[0]}")/..
-SDK_VERSION=v0.3.0
+SDK_VERSION=${SDK_VERSION:=v0.19.1}
 
 # Get operator-sdk binary.
 wget -O /tmp/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/${SDK_VERSION}/operator-sdk-${SDK_VERSION}-x86_64-linux-gnu && chmod +x /tmp/operator-sdk
```

Comment:
Also, we may want to migrate towards having a dedicated built root Dockerfile if we're requiring operator-sdk being installed when running e2e tests, rather than curling a specific release binary when running this script.